test/testsuites/packagestests.go

```diff
@@ -10,10 +10,17 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// TestCscWithNonExistingPackage tests that a csc with a non-existing package
+// InvalidPackages is a test suite that ensures that package related issues are handled
+// correctly.
+func InvalidPackages(t *testing.T) {
+	t.Run("csc-with-non-existing-package", testCscWithNonExistingPackage)
+	t.Run("opsrc-with-conflicting-packages", testConflictingPackages)
```

Comment:
I don't see a function called `testConflictingPackages()`. How is this working?

Reply:
Thanks for pointing this out @aravindhp, I had noticed after I made the original PR but decided not to update the PR and kick off an additional test because CI/CD was failing due to high load on the infrastructure at the time. I should have added a comment after making the change.